httplib.h

```diff
@@ -3890,7 +3890,8 @@ inline bool parse_multipart_boundary(const std::string &content_type,
                                      std::string &boundary) {
   auto pos = content_type.find("boundary=");
   if (pos == std::string::npos) { return false; }
-  boundary = content_type.substr(pos + 9);
+  auto end = content_type.find(';', pos);
+  boundary = content_type.substr(pos + 9, end - pos - 9);
```

Comment:
Could it be easier to understand?
```c++
auto beg = pos + 9;
boundary = content_type.substr(beg, end - beg);
```

Reply:
Yes, totally. I agree. I will make the modification tomorrow. Thanks for the suggestion.

Reply:
Done, and pushed on my branch.